stable/dask/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+name: dask
+version: 0.17.0
```

Comment:
Currently I'm pinning the version of the chart to the version of the main software package.  I'm also happy to go directly to 1.0.0 and follow semver if preferred

Reply:
Add `appVersion` which should be `0.17.0`. The chart is versioned independently.